Source/WebCore/layout/formattingContexts/inline/InlineContentConstrainer.cpp

```diff
@@ -187,13 +179,21 @@ void InlineContentConstrainer::checkCanConstrainInlineItems()
     }
 }
 
-std::optional<size_t> InlineContentConstrainer::buildLineWithHyphenationFrom(size_t)
+InlineContentConstrainer::EntryPretty InlineContentConstrainer::buildOneLineLayout(InlineItemRange layoutRange, InlineLayoutUnit idealLineWidth, EntryPretty lastValidEntry)
```

Comment:
I should refrain from commenting on names but `buildOneLineLayout` sounds a bit odd. We don't normally **_build_** line **_layout_**.